examples/bayesian_linear_regression.py

```diff
@@ -38,35 +38,35 @@ class LinearModel:
     def __init__(self, lik_variance=0.01, prior_variance=0.01):
         self.lik_variance = lik_variance
         self.prior_variance = prior_variance
-        self.num_vars = 2
+        self.n_vars = 2
 
     def log_prob(self, xs, zs):
         """Returns a vector [log p(xs, zs[1,:]), ..., log p(xs, zs[S,:])]."""
         x, y = xs['x'], xs['y']
         log_prior = -self.prior_variance * tf.reduce_sum(zs*zs, 1)
-        # broadcasting to do (x*W) + b (n_data x n_minibatch - n_minibatch)
+        # broadcasting to do (x*W) + b (n_minibatch x n_samples - n_samples)
         W = tf.expand_dims(zs[:, 0], 0)
         b = zs[:, 1]
         mus = tf.matmul(x, W) + b
-        # broadcasting to do mus - y (n_data x n_minibatch - n_data x 1)
+        # broadcasting to do mus - y (n_minibatch x n_samples - n_minibatch x 1)
         y = tf.expand_dims(y, 1)
         log_lik = -tf.reduce_sum(tf.pow(mus - y, 2), 0) / self.lik_variance
         return log_lik + log_prior
 
-def build_toy_dataset(n_data=40, noise_std=0.1):
+def build_toy_dataset(n_minibatch=40, noise_std=0.1):
```

Comment:
This one refers to the number of data points for building the toy data set. So maybe something like `N`?